{"uid":"4f75f53899e51223","name":"Авторизация пользователя с валидным логин и паролем через API","fullName":"tests.test_authorization.TestAuthUser#test_api_auth_valid_username_and_valid_password","historyId":"1396a5305a37c317f3d46e05e4e7128e","time":{"start":1738871535420,"stop":1738871572989,"duration":37569},"description":"При проверке используются валидный логин и пароль пользователя Aleska","descriptionHtml":"<p>При проверке используются валидный логин и пароль пользователя Aleska</p>\n","status":"failed","statusMessage":"AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.","statusTrace":"self = <base.base_assertion.Assertion object at 0x7fb57c233f20>\npage_url = 'https://staging.round.zone/rubric', timeout = 10\n\n    def page_is_opened(self, page_url, timeout=10):\n        \"\"\"\n        Функция проверки, что запрашиваемая страница открыта\n    \n        :param page_url:\n            url страницы\n        :param timeout:\n            время ожидания в секундах\n        \"\"\"\n        try:\n            self.element_is_not_visible(AllPageLocators.locators['Спиннер загрузки'])\n>           self.wait.until(EC.url_to_be(page_url), timeout)\n\nbase/base_assertion.py:60: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"fe79396aa69dabceaedc80378d98e78f\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7fb57cad3740>\nmessage = 10\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: 10\n\n../local/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_authorization.TestAuthUser object at 0x7fb57d6901a0>\n\n    @allure.title('Авторизация пользователя с валидным логин и паролем через API')\n    @allure.description('При проверке используются валидный логин и пароль пользователя Aleska')\n    @allure.severity('Critical')\n    @pytest.mark.regression\n    def test_api_auth_valid_username_and_valid_password(self):\n        with allure.step('Открыть страницу логина'):\n            self.api_authorization.user(self.User.LOGIN, self.User.PASSWORD)\n    \n        with allure.step('Проверяем, что открывается страница рубрик'):\n>           self.assertion.page_is_opened(self.rubric_page.PAGE_URL)\n\ntests/test_authorization.py:150: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <base.base_assertion.Assertion object at 0x7fb57c233f20>\npage_url = 'https://staging.round.zone/rubric', timeout = 10\n\n    def page_is_opened(self, page_url, timeout=10):\n        \"\"\"\n        Функция проверки, что запрашиваемая страница открыта\n    \n        :param page_url:\n            url страницы\n        :param timeout:\n            время ожидания в секундах\n        \"\"\"\n        try:\n            self.element_is_not_visible(AllPageLocators.locators['Спиннер загрузки'])\n            self.wait.until(EC.url_to_be(page_url), timeout)\n        except TimeoutException:\n            current_url = self.driver.current_url\n>           raise AssertionError(f'Ожидалась страница {page_url}, но открылась {current_url}. '\n                                 f'Страница не загрузилась за {timeout} секунд.')\nE           AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.\n\nbase/base_assertion.py:63: AssertionError","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1738870943195,"stop":1738870943196,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1738871535003,"stop":1738871535420,"duration":417},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1738871535420,"stop":1738871535420,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"При проверке используются валидный логин и пароль пользователя Aleska","status":"failed","statusMessage":"AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.","statusTrace":"self = <base.base_assertion.Assertion object at 0x7fb57c233f20>\npage_url = 'https://staging.round.zone/rubric', timeout = 10\n\n    def page_is_opened(self, page_url, timeout=10):\n        \"\"\"\n        Функция проверки, что запрашиваемая страница открыта\n    \n        :param page_url:\n            url страницы\n        :param timeout:\n            время ожидания в секундах\n        \"\"\"\n        try:\n            self.element_is_not_visible(AllPageLocators.locators['Спиннер загрузки'])\n>           self.wait.until(EC.url_to_be(page_url), timeout)\n\nbase/base_assertion.py:60: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"fe79396aa69dabceaedc80378d98e78f\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7fb57cad3740>\nmessage = 10\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: 10\n\n../local/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_authorization.TestAuthUser object at 0x7fb57d6901a0>\n\n    @allure.title('Авторизация пользователя с валидным логин и паролем через API')\n    @allure.description('При проверке используются валидный логин и пароль пользователя Aleska')\n    @allure.severity('Critical')\n    @pytest.mark.regression\n    def test_api_auth_valid_username_and_valid_password(self):\n        with allure.step('Открыть страницу логина'):\n            self.api_authorization.user(self.User.LOGIN, self.User.PASSWORD)\n    \n        with allure.step('Проверяем, что открывается страница рубрик'):\n>           self.assertion.page_is_opened(self.rubric_page.PAGE_URL)\n\ntests/test_authorization.py:150: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <base.base_assertion.Assertion object at 0x7fb57c233f20>\npage_url = 'https://staging.round.zone/rubric', timeout = 10\n\n    def page_is_opened(self, page_url, timeout=10):\n        \"\"\"\n        Функция проверки, что запрашиваемая страница открыта\n    \n        :param page_url:\n            url страницы\n        :param timeout:\n            время ожидания в секундах\n        \"\"\"\n        try:\n            self.element_is_not_visible(AllPageLocators.locators['Спиннер загрузки'])\n            self.wait.until(EC.url_to_be(page_url), timeout)\n        except TimeoutException:\n            current_url = self.driver.current_url\n>           raise AssertionError(f'Ожидалась страница {page_url}, но открылась {current_url}. '\n                                 f'Страница не загрузилась за {timeout} секунд.')\nE           AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.\n\nbase/base_assertion.py:63: AssertionError","steps":[{"name":"Открыть страницу логина","time":{"start":1738871535420,"stop":1738871539891,"duration":4471},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Проверяем, что открывается страница рубрик","time":{"start":1738871539891,"stop":1738871572988,"duration":33097},"status":"failed","statusMessage":"AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.\n","statusTrace":"  File \"/usr/workspace/tests/test_authorization.py\", line 150, in test_api_auth_valid_username_and_valid_password\n    self.assertion.page_is_opened(self.rubric_page.PAGE_URL)\n  File \"/usr/workspace/base/base_assertion.py\", line 63, in page_is_opened\n    raise AssertionError(f'Ожидалась страница {page_url}, но открылась {current_url}. '\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1738871572999,"stop":1738871573208,"duration":209},"status":"passed","steps":[],"attachments":[{"uid":"f1fad8072ea59b94","name":"page_source","source":"f1fad8072ea59b94.html","type":"text/html","size":31587},{"uid":"3060a19639baafd6","name":"screenshot","source":"3060a19639baafd6.png","type":"image/png","size":60710},{"uid":"5ebfb106cd57706e","name":"browser_logs","source":"5ebfb106cd57706e.txt","type":"text/plain","size":7069}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"feature","value":"Авторизация пользователя"},{"name":"tag","value":"regression"},{"name":"tag","value":"authorization"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_authorization"},{"name":"subSuite","value":"TestAuthUser"},{"name":"host","value":"f1e44e101a82"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_authorization"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":5,"skipped":0,"passed":54,"unknown":0,"total":60},"items":[{"uid":"533faf4936596b25","status":"passed","time":{"start":1738864905043,"stop":1738864909830,"duration":4787}},{"uid":"d1210528fc8abb87","status":"passed","time":{"start":1738268556054,"stop":1738268563498,"duration":7444}},{"uid":"d5c05ab770a45be7","status":"passed","time":{"start":1738098995650,"stop":1738099002398,"duration":6748}},{"uid":"6832413afebd900d","status":"passed","time":{"start":1738005006944,"stop":1738005014775,"duration":7831}},{"uid":"cab141389226d882","status":"passed","time":{"start":1738002338386,"stop":1738002343888,"duration":5502}},{"uid":"49d98fd51dc6f54d","status":"passed","time":{"start":1737927950950,"stop":1737928014729,"duration":63779}},{"uid":"6ef344d5e3d72864","status":"passed","time":{"start":1737842612840,"stop":1737842622282,"duration":9442}},{"uid":"ef533e4ad6464f74","status":"passed","time":{"start":1737831903986,"stop":1737831910555,"duration":6569}},{"uid":"daeec1135f684bb","status":"passed","time":{"start":1737799238921,"stop":1737799245727,"duration":6806}},{"uid":"f38f654567bd9923","status":"passed","time":{"start":1737757514460,"stop":1737757519645,"duration":5185}},{"uid":"3aaa4fee4357e6b7","status":"passed","time":{"start":1737744472310,"stop":1737744477434,"duration":5124}},{"uid":"77dc845f6087a0e5","status":"passed","time":{"start":1737652253427,"stop":1737652260995,"duration":7568}},{"uid":"769fcedd8170f81c","status":"passed","time":{"start":1737569875159,"stop":1737569883062,"duration":7903}},{"uid":"4b730976def764fc","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568960991,"stop":1737568960991,"duration":0}},{"uid":"49b91a53ca6d85dd","status":"passed","time":{"start":1734034286316,"stop":1734034291636,"duration":5320}},{"uid":"af5f1a9ca18d5673","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733936961309,"stop":1733936996315,"duration":35006}},{"uid":"24167f51472f75a1","status":"passed","time":{"start":1733865448920,"stop":1733865454455,"duration":5535}},{"uid":"955ac0d377cf726c","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733859897987,"stop":1733859933844,"duration":35857}},{"uid":"a10777b89236b2ac","status":"passed","time":{"start":1733856557604,"stop":1733856563152,"duration":5548}},{"uid":"856c272798216cda","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733771397851,"stop":1733771433163,"duration":35312}}]},"tags":["authorization","regression"]},"source":"4f75f53899e51223.json","parameterValues":[]}